numpy/lib/tests/test_shape_base.py

```diff
@@ -55,9 +61,45 @@ class MinimalSubclass(np.ndarray):
             pass
         a = np.ones((6, 3)).view(MinimalSubclass)
         res = apply_along_axis(np.sum, 0, a)
-        assert isinstance(res, MinimalSubclass)
+        assert_(isinstance(res, MinimalSubclass))
         assert_array_equal(res, np.array([6, 6, 6]).view(MinimalSubclass))
 
+    def test_0d_array(self):
+        class MinimalSubclass(np.ndarray):
```

Comment:
How about `test_0d_array(self, cls=np.ndarray)`, and having the subclass tests call that with a subclass?